modules/local/drop_config_runAE.nf

```diff
@@ -49,12 +53,20 @@ process DROP_CONFIG_RUN_AE {
         --drop_module AE \\
         --genome_assembly $genome_assembly \\
         --drop_group_samples $drop_group \\
+        --drop_other_group_samples $drop_other_group_samples \\
         --padjcutoff ${drop_padjcutoff_ae} \\
         $zscorecutoff \\
         --output config.yaml
 
     snakemake aberrantExpression --cores ${task.cpus} --rerun-triggers mtime $args
 
+    if [[ !skip_export_counts_drop ]]; then
```

Comment:
I don't think this will work as intended.

I am testing running it as such now:

```
if [[ "${skip_export_counts_drop}" == "false" ]]; then
```

Reply:
I have just finished running it and it did work for us. Let me know how it goes for you.

Reply:
Hmm. Have you tried running with the setting turned both on and off?

This is bash here, i.e. you would need to prefix the variable with `$` for the variable to be recognized.

Similarly to how you are doing below:

```
--genome_assembly $genome_assembly \\
```

Or am I missing something?

Reply:
I tried to find the old error messages, but did only find my crashes resolved by `mkdir -p`. Seems it shouldn't work, but not sure 🤔 I'll let you know if I run into this again, that time with actual error messages.

Reply:
I will give it another try and get back to you

Reply:
You are right, the folder appears even when the skip = true. I have tested your suggestion and it works, so I will update the code. Thanks for noticing :D